src/components/MenuButton/src/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StylesProvider } from '@material-ui/core/styles';
```

Comment:
Import the StylesProvider from `@material-ui/core` (PS we don't currently need the styles provider since there's no MUI component being wrapped.)